test/active_model_serializers/register_jsonapi_renderer_test_isolated.rb

```diff
@@ -58,21 +63,6 @@ def test_jsonapi_parser_not_registered
       assert_nil parsers[Mime[:jsonapi]]
     end
 
-    def test_jsonapi_renderer_not_registered
-      expected = {
-        'data' => {
-          'attributes' => {
-            'name' => 'Johnny Rico'
-          },
-          'type' => 'users'
-        }
-      }
-      payload = '{"data": {"attributes": {"name": "Johnny Rico"}, "type": "authors"}}'
-      headers = { 'CONTENT_TYPE' => 'application/vnd.api+json' }
-      post '/render_with_jsonapi_renderer', params: payload, headers: headers
-      assert expected, response.body
```

Comment:
the problem is that it raises an exception while trying to create a new author since params are not parsed correctly without this.  So, I can setup another api method.

I think i have a way to change controller to test this case.  Let me take a shot.